webapi/src/main/scala/org/knora/webapi/slice/common/api/AuthorizationRestService.scala

```diff
@@ -101,6 +110,16 @@ final case class AuthorizationRestServiceLive() extends AuthorizationRestService
       s"You are logged in with username '${user.username}', but only a system administrator, system user or project administrator has permissions for this operation."
     checkActiveUser(user, isSystemAdminSystemUserOrProjectAdmin(project), msg)
   }
+
+  override def ensureSystemAdminOrProjectAdminInAnyProject(requestingUser: User): IO[ForbiddenException, Unit] = {
+    lazy val msg = s"ProjectAdmin or SystemAdmin permissions are required."
```

Comment:
Why would this be a lazy val? 😁

Reply:
No reason now, before actually used String interpolation to include some user information into the message, this was only necessary in case of an error.

```suggestion
    val msg = "ProjectAdmin or SystemAdmin permissions are required."
```